pages/portfolio.html

```diff
@@ -4,9 +4,29 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Portfolio</title>
+  <link href="../stylesheets/style.css" rel="stylesheet">
 </head>
 <body>
-  
+  <header>
+    <h1>Kiana Shui</h1>
+    <nav>
+      <a href="index.html">Home</a>
+      <a href="about.html">About</a>
+      <a href="portfolio.html">Portfolio</a>
+    </nav>
+  </header>
+  <main>
+    <h2>Projects</h2>
+    <p>Viewing Party: Compare everyone's watched movie list, favorites list, and watchlist.</p>
+    <a href="https://github.com/kianashui/viewing-party"><img alt="Viewing Party" src="../images/video_clipart.png"></a>
+    <p>Swap Meet: Swap items between different vendors' inventories.</p>
+    <a href="https://github.com/kianashui/swap-meet"><img alt="Swap Meet" src="../images/swap_clipart.png"></a>
+    <p>Task List API: Keep track of tasks and goals, sending messages to Slack when a task is completed.</p>
+    <a href="https://github.com/kianashui/task-list-api"><img alt="Task List API" src="../images/task_clipart.png"></a>
```

Comment:
It might have made sense to make the name of each project a `h3` so that they wind up called out structurally.